### PR TITLE
Refs #23526 - add a note that we don't check for ElasticSearch anymore

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Review by ekohl — APPROVED** [review]

File: plugins/katello/nightly/release_notes/release_notes.md [hunk 1]
```diff
@@ -12,5 +12,7 @@ For the full release notes, see the [Changelog](https://github.com/Katello/katel
 
 ## Deprecation Warnings
 
+* Katello used to use ElasticSearch, which was removed in Katello 3.0. After that the Installer would check and warn users if they would still have ElasticSearch installed. This check is now gone. If you still have ElasticSearch installed, please remove the RPMs (`ruby193-rubygem-tire tfm-rubygem-tire elasticsearch sigar-java sigar snappy-java lucene4-contrib lucene4`) and drop the database (`/var/lib/elasticsearch`).
+
```
Comment (mmoll) [inline plugins/katello/nightly/release_notes/release_notes.md]: As a user I'd be unsure, what this means. Delete the directory? If yes, "...drop the database by deleting..." would be better

Comment (evgeni) [reply plugins/katello/nightly/release_notes/release_notes.md]: You are not a user! ;-) (fixing)

**Comment by mmoll** [comment]
> You are not a user! ;-)

correct, I guess "drop the database" might already raise eyebrows there ;)

**Comment by mmoll** [comment]
merged, спасибо Женя!

**Pull request merged**